src/TraceEvent/NativeDlls.cs

```diff
@@ -15,7 +15,9 @@ class NativeDlls
     /// <param name="simpleName"></param>
     public static void LoadNative(string simpleName)
     {
-        var thisDllDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().ManifestModule.FullyQualifiedName);
+        // When TraceEvent is loaded as embedded assembly the executing assembly is mscorlib which points to the wrong path.
```

Comment:
By the way the publicly used KernelTraceControl.dll does not work on Windows 7 because it is linked with ApiSets agains Win8? It fails with  

`Application popup: LiveExceptionMonitor.exe - Entry Point Not Found : The procedure entry point ReleaseSRWLockExclusive could not be located in the dynamic link library api-ms-win-core-synch-l1-2-0.dll. 
`
That should be fixed as well. 
As feature request I would like to use "Circular Kernel Context Logger" as my second trace logger to get out of the way of procmon, procexp, ... and "real" profiling. That would also be a big help. This is a big help on Win7 where officially there is only one kernel logger allowed.